functional/vcpus/default_vcpus_test.sh

```diff
@@ -29,6 +30,11 @@ if [ "$TEST_INITRD" == "yes" ]; then
 	exit 0
 fi
 
+if [ "${NAME}" == "Ubuntu" ] && [ "$(echo "${VERSION_ID} >= 22.04" | bc -q)" == "1" ]; then
+	echo "Skip vcpu test is not working"
```

Comment:
Whats the error you are seeing with vcpu tests?

Reply:
@amshinde 
12:45:42 Running ctr integration tests with vcpus
12:45:43 quay.io/prometheus/busybox:latest: resolving      |[32m[0m--------------------------------------| 
12:45:43 elapsed: 0.1 s                     total:   0.0 B (0.0 B/s)                                         
12:45:43 quay.io/prometheus/busybox:latest:                                                resolved       |[32m++++++++++++++++++++++++++++++++++++++[0m| 
12:45:43 index-sha256:918fd39802562ff02d4232970ee50a11f7884a2699adfefdb500fa61e466c0cc:    done           |[32m++++++++++++++++++++++++++++++++++++++[0m| 
12:45:43 manifest-sha256:6594015d3ec4a8fa22c96d419491c2064462e409dd59b68101297de59f83f55d: done           |[32m++++++++++++++++++++++++++++++++++++++[0m| 
12:45:43 layer-sha256:7fe869475116a421aa5ff34a0dd1c4015e4b04f2b0b86682cea21b9695ff77f5:    done           |[32m++++++++++++++++++++++++++++++++++++++[0m| 
12:45:43 config-sha256:342b02f8cb5f577c5868ed34f8d8901be614b67cf6e82817fadeb0e612292c80:   done           |[32m++++++++++++++++++++++++++++++++++++++[0m| 
12:45:43 layer-sha256:19d511225f94f9b5cbf3836eb02b5273c01b95da50735742560e3e45b8c8bfcc:    done           |[32m++++++++++++++++++++++++++++++++++++++[0m| 
12:45:43 elapsed: 0.2 s                                                                    total:   0.0 B (0.0 B/s)                                         
12:45:43 unpacking linux/amd64 sha256:918fd39802562ff02d4232970ee50a11f7884a2699adfefdb500fa61e466c0cc...
12:45:43 done: 11.468708ms	
12:45:43 ctr: failed to create shim: write /sys/fs/cgroup/vc/cgroup.procs: device or resource busy: unknown